mungegithub/mungers/blunderbuss.go

```diff
@@ -68,6 +70,7 @@ func (b *BlunderbussMunger) EachLoop() error { return nil }
 // AddFlags will add any request flags to the cobra `cmd`
 func (b *BlunderbussMunger) AddFlags(cmd *cobra.Command, config *github.Config) {
 	cmd.Flags().BoolVar(&b.BlunderbussReassign, "blunderbuss-reassign", false, "Assign PRs even if they're already assigned; use with -dry-run to judge changes to the assignment algorithm")
+	cmd.Flags().BoolVar(&b.UseReviewers, "blunderbuss-reviewers", false, "Use \"reviewers\" rather than \"approvers\" for blunderbuss")
```

Comment:
Do you mean "standardized on all mungers" ?

The question is "Do we have two flags, or do we have one in `mungegithub.go`?". Both solutions are kind of gross. Also, I don't think it really matters, because ideally this flag will be removed quickly, and is just there to ease transitioning.